internal/provider/custom_policy.go

```diff
@@ -63,6 +66,10 @@ func (r *customPolicyResource) Schema(ctx context.Context, req resource.SchemaRe
 				MarkdownDescription: "Mondoo space identifier. If there is no space ID, the provider space is used.",
 				Optional:            true,
 			},
+			"scope_mrn": schema.StringAttribute{
```

Comment:
Since we have now both, Space ID and Scope MRN, we should add a `ConflictsWith` entry to prevent users to provide both. (docs https://developer.hashicorp.com/terraform/plugin/sdkv2/schemas/schema-methods)